tests/conftest.py

```diff
@@ -9,3 +9,13 @@ def base_clumper():
         for i, c in enumerate("abcdefghijklmnopqrstuvwxyz")
     ]
     return Clumper(data)
+
+
+@pytest.fixture(scope="module")
+def single_local_jsonl_file_path():
```

Comment:
Adding a fixture for a single file path that is only used in a single test is a bit of overkill. If multiple tests in a single file use it then it makes sense to add a fixture for a file level. But in this case we're dealing with fixtures that are used by one test in one file. Best to remove the fixtures here and just use the path directly in the test.

Reply:
Same for the fixture below.

Reply:
My logic was: Maybe I want to add more tests in the future, and I don't want to keep on copy-pasting the file path. Do you think that this is premature optimization?

Reply:
I think it is. But it's a detail that I can easily adres later down the line.